Automattic/jetpack
# Protect: Add brute force protection toggle

## Proposed changes:

* Adds `brute_force_protection` boolean property to the WAF config.
* Adds a toggle to the firewall settings page to enable/disable brute force protection.

### Other information:


## Jetpack product discussion



1201069996155224-as-1203716242675366

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






Starting with a site that has Jetpack Protect running this branch, and no other Jetpack plugins installed:

* Go to the Protect admin page
* Validate that brute force protection is enabled by default.
* Validate that the login screen presents a math problem - may take a few failed attempts to show.
* Disable brute force protection in Protect.
* Validate that the login screen no longer includes the math problem.
* Re-enable brute force protection.
* Validate that the login screen presents a math problem - may take a few failed attempts to show.

## Screenshots:

<img width="782" alt="Screenshot 2023-03-09 at 10 12 49 AM" src="https://user-images.githubusercontent.com/10933065/224109075-b7078134-8747-4ad1-bea3-ea535670e16f.png">
